common_knowledge/GitBranching.md

```diff
@@ -0,0 +1,45 @@
+
+# Git Branching
+
+We use a model something close to [Trunk-based development](https://www.atlassian.com/continuous-delivery/continuous-integration/trunk-based-development) 
+where we try to merge code to master often and send in small pull requests.
+
+We use the standard/default merge strategy for Git which avoids helps avoid
+merge conflicts so you can merge master and smaller (possibly stacked) PRs into
+your development branch often.
+
+# Key Suggestions
+
+## Commit early, commit often
+
+Make sure to commit your code early and often. This allows for smaller and
+discrete integration tests but also reduces the chance that your code will cause
+a merge conflict later.
+
+## Conventional Commits
+
+Use [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) in your PR title.
+
+## Send Smaller PRs
```

Comment:
agree, but some exceptions are when refactors touch every/multiple files

Reply:
I agree with @mzparacha. We should add an 'Exception' section detailing that if a large PR _is_ required the PR description should include justification.